bika/health/upgrade/v01_01_003.py

```diff
@@ -31,7 +35,72 @@ def upgrade(tool):
     # Ensure health's skins have always priority over core's
     setup = portal.portal_setup
     setup.runImportStepFromProfile(profile, "skins")
+    setup.runImportStepFromProfile(profile, 'workflow')
+    setup.runImportStepFromProfile(profile, 'propertiestool')
+
+    add_doctor_action_for_client(portal)
+    ut.addIndexAndColumn('portal_catalog', 'allowedRolesAndUsers', 'FieldIndex')
+    ut.addIndex('portal_catalog', 'getPrimaryReferrerUID', 'FieldIndex')
+    ut.refreshCatalogs()
+
+    update_permissions_clients(portal)
 
     logger.info("{0} upgraded to version {1}".format(product, version))
 
     return True
+
+
+def add_doctor_action_for_client(portal):
+    # Add doctor action for client portal_type programmatically
```

Comment:
Instead of a comment we could put the documentation of the function in a docstring?